crates/anvil/src/eth/api.rs

```diff
@@ -1352,6 +1356,15 @@ impl EthApi {
         Ok(self.backend.get_blob_by_tx_hash(hash)?)
     }
 
+    /// Handler for RPC call: `anvil_getBlobSidecarsByBlockId`
+    pub fn anvil_get_blob_sidecars_by_block_id(
```

Comment:
we can mention here that this returns the beacon api format and reference the doc